analyses/molecular-subtyping-EPN/02_ependymoma_generate_all_data.R

```diff
@@ -87,7 +98,21 @@ breakpoint_density_sv <- breakpoint_density_sv %>%
 # Reading consensus focal CN results from analyses
 focal_cn_gene <- read_tsv(focal_gene_cn)
 focal_cn_gene_CDKN2A <- focal_cn_gene %>%
-  filter(gene_symbol == "CDKN2A") 
+  filter(gene_symbol == "CDKN2A" & status %in% c("amplification", "gain", "loss")) %>%
+  distinct(biospecimen_id, .keep_all = TRUE) %>%
+  column_to_rownames('biospecimen_id')
+focal_cn_gene_MYCN <- focal_cn_gene %>%
+  filter(gene_symbol == "MYCN" & status %in% c("amplification", "gain", "loss")) %>%
```

Comment:
@rjcorb Is there any reason that the only things you are selecting are `gain` and `loss` when there are other non-amplified identifiers in the column such as `deep deletion` and `neutral`?

Also, why is NF2 included in the focal-cn check? In the ticket only NF2 mutation is optional for subtyping `EPN, SP` and `EPN, SP-MYCN` and not focal-cn.  

```
> read_tsv("../../data/consensus_wgs_plus_cnvkit_wxs_autosomes.tsv.gz") %>% filter(gene_symbol == "CDKN2A") %>% select(status, gene_symbol) %>% distinct()
 status    gene_symbol
 <chr>     <chr>    
1 neutral    CDKN2A   
2 loss     CDKN2A   
3 gain     CDKN2A   
4 deep deletion CDKN2A   
5 amplification CDKN2A
```

Reply:
@atlas4213 there are currently instances of multiple status calls for the same gene ID in the focal CN file. This is currently being addressed in [Ticket #436](https://github.com/PediatricOpenTargets/ticket-tracker/issues/436) and [Ticket #437](https://github.com/PediatricOpenTargets/ticket-tracker/issues/437). Once this is resolved and there is a single call for each gene we can modify the code. Among EPN samples, duplicate CN calls consisted of neutral/loss, neutral/gain, loss/deep deletion, and loss/loss calls. So filtering out 'neutral' and 'deep deletion' calls resolved this issue for now, but it will need to be revisited once those tickets are resolved. As of right now it does not result in any incorrect subtyping of EPN samples.

Reply:
@jharenza recommended adding NF2 focal CN data as well to look at co-occurence of NF2 SNVs and CNVs.